docs/orchestration/concepts/task-concurrency-limiting.md

```diff
@@ -39,7 +39,7 @@ Once you have tagged your various tasks and [registered your Flow(s)](flows.html
 
 ### UI
 
-To set Task tag concurrency limits from the UI, go to Prefect Cloud and navigate to Team Settings -> Task Concurrency Limits.
+To set task concurrency limits from the UI, go to Prefect Cloud and navigate to Team Settings -> Task Concurrency Limits.
```

Comment:
I don't see the benefit in removing the 'tag' descriptor here and elsewhere - the limits are based on tags